cmd/cloud/deploy_test.go

```diff
@@ -23,7 +23,7 @@ func TestDeployImage(t *testing.T) {
 		return nil
 	}
 
-	deployImage = func(path, deploymentID, wsID, pytest, envFile string, prompt bool, client astro.Client) error {
+	deployImage = func(path, deploymentID, wsID, pytest, envFile string, imageName string, prompt bool, client astro.Client) error {
```

Comment:
Should this change be?

```suggestion
	deployImage = func(path, deploymentID, wsID, pytest, envFile, imageName string, prompt bool, client astro.Client) error {
```

Reply:
yes thanks